1st-page/dogs.js

```diff
@@ -0,0 +1,65 @@
+function dogPic(name, breed, price) {
+    alert(name, breed, price);   
+}
+
+//function murphyImg() {
+//    let murphy = alert('Murphy, Adult breed, $123');
+//}
+
+//function violetImg() {
+//    let violet = alert('Violet, puppy breed, $123');
+//}
+
+function piperImg() {
```

Comment:
While this approach works, it is inefficient. A better approach would be to have one function for all dog image clicks, and instead pass it parameters to alert. Similar to the approach for the onDogPrice() function below.